src/guide/admin/clients.md

```
@@ -1,32 +1,32 @@
-# Robots
+# Clients
 ::: info Notice
 In order to manage your identity-providers, nodes, ... you need to get an organization. Please contact us,
 such we can provide you a realm in which you can manage all your nodes, users and identity providers.
 For more details on realms click [here](/guide/admin/realms).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)